my-mini-compiler/CMakeLists.txt

```
@@ -8,8 +8,10 @@ set(CMAKE_C_STANDARD 11)
 # Add include directory (this will be needed to add your tokens to your lexer)
 include_directories(${PROJECT_SOURCE_DIR}/phase1-w25/include)
 
-#find all the .c files in the src directory (don't put random .c files in the src directory or they will be compiled brrr)
-file(GLOB_RECURSE SOURCES "phase1-w25/src/**/*.c")
+set(CMAKE_C_FLAGS "-g -Werror -Wall -Wextra -Wpedantic") # strict build flags
```

I think you should be using add_compile_options instead of set, but if this was intentional feel free to ignore 

```suggestion
add_compile_options(-g -Werror -Wall -Wextra -Wpedantic) # strict build flags
```